docs/docs/tailwind-css.md

```diff
@@ -149,6 +149,112 @@ plugins: [
 **Note:** Optionally you can add a corresponding configuration file (by default it will be `tailwind.config.js`).
 If you are adding a custom configuration, you will need to load it after `tailwindcss`.
 
+### 4. Add custom css/scss files
+
+**Note**: This approach is not needed if you chose CSS-in-JS above, as you can already nest styles and @apply rules directly from your .js files.
+
+In case you need to create custom classes for elements for nested selectors, or for overriding external packages, you can create your own css/scss files.
+
+1. Create a new file and import your tailwind directives.
+
+This will be your 'master' css file, which you will import all other css within.
+
+```css:title=src/css/index.css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+**Note**: if using scss (or another supported language, rename files/folders appropriately).
+
+2. Import any custom css files or add any custom css you need (optional)
+
+```css:title=src/css/index.css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import popup.css
+
+body {
+  @apply bg-purple-200;
+}
+```
+
+3. Apply these additional styles to the browser
+
+In `gatsby-browser.js` add an import rule for your tailwind directives and custom css so that they are accounted for in build.
+
+```js:title=gatsby-browser.js
+import './src/css/index.css'
+```
+
+### 5. Purging your css
+Now we've fully configured tailwindcss, we want to make sure that only the classes we need are delivered to the browser. By default, tailwind is a very large library because it includes every combination of every class you might think of. Most of these you won't need, so we use PurgeCSS to remove any unused classes. 
+
+**Note**: By default, PurgeCSS only runs on the build command as it is a relatively slow process. The development server will include all tailwind classes, so it's highly recommended you test on a build server before deploying.
+
+From v1.4.0 onwards PurgeCSS is built into TailwindCSS, but the approaches needed are very similar.
+
+#### 1.4.0 and above
+
+In 1.4.0 you can purge your css directly from your tailwind config. You simply need to provide an array of strings telling it which files to process.
+
+```js:title=tailwind.config.js
+module.exports = {
+  purge: [
+    './src/**/*.js',
+    './src/**/*.jsx',
+    './src/**/*.ts',
+    './src/**/*.tsx',
+  ],
+  theme: {},
+  variants: {},
+  plugins: [],
+}
+```
+
+Full documentation on this can now be found on the Tailwind site - [Tailwind PurgeCSS documentation](https://tailwindcss.com/docs/controlling-file-size/#app)
+
+#### Older versions
```

Comment:
maybe we should only include docs for the newest version? I'd assume people wanting purgecss would either upgrade or find it directly.

Reply:
I can remove. I added it as v1.4.0 only came out a week and a half ago, so most people will still be using the old approach. But you're right, it makes sense to direct people to the officially supported/recommended method.

Reply:
@gatsbyjs/learning 
Updated as requested, removing all but the bare minimum support for older versions of TailwindCSS/PurgeCSS.